app/allocation/controllers/assign-to-allocation.js

```diff
@@ -0,0 +1,12 @@
+function assignToAllocation(req, res) {
+  const { allocation } = req
+  const movesWithoutProfile = allocation.moves.filter(move => !move.profile)
+
+  if (movesWithoutProfile.length) {
+    return res.redirect(`/move/${movesWithoutProfile[0].id}/assign`)
+  }
+
+  res.redirect(`/allocation/${allocation.id}`)
```

Comment:
Does the page handle no more moves left? I'm sure it does, I just don't think I've seen that scenario :D

Reply:
Yep, that's what this catch is for. 

So when all moves are filled, this will catch someone clicking the "Add person" button in an old tab. It redirects back to the allocation (which will now show as filled) rather than `/move/undefined/assign` where they'd get a `404`.